geonode/base/models.py

```diff
@@ -804,7 +822,7 @@ class Link(models.Model):
         * OGC:WFS: for WFS service links
         * OGC:WCS: for WCS service links
     """
-    resource = models.ForeignKey(ResourceBase)
+    resource = models.ForeignKey(ResourceBase, blank=True, null=True)
```

Comment:
Same as above.

It is worth notice that this change does not impact GeoNode at all.